feature/staticroutes/staticroutes.go

```diff
@@ -0,0 +1,69 @@
+/*
+ Copyright 2022 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+// Package networkinstance implements the Config Library for NetworkInstance
+// feature.
+package networkinstance
+
+import (
+	"errors"
+
+	"github.com/openconfig/featureprofiles/yang/oc"
+	"github.com/openconfig/ygot/ygot"
+	"strconv"
+)
+
+// NetworkInstance struct stores the OC attributes.
+type NetworkInstance struct {
```

Comment:
type StaticRoute struct

Reply:
changed. Have removed this for now and will re-add it back again